modules/rdf.xqm

```diff
@@ -203,16 +208,122 @@ as element(rdf:RDF) {
 
   let $in_corpus := <dracon:in_corpus rdf:resource="{$collection-uri}"/>
   let $play-external-id := <owl:sameAs rdf:resource="{$wikidata-play-uri}"/>
+  
+  (: CIDOC-Stuff :)
+  let $creation-uri := $play-uri || "/creation"
+  let $created-by :=  <crm:P94i_was_created_by rdf:resource="{$creation-uri}"/>
+  
+  let $creation-activity := 
+    <rdf:Description rdf:about="{$creation-uri}">
+        <rdf:type rdf:resource="http://www.cidoc-crm.org/cidoc-crm/E65_Creation"/>
+        <crm:P94_has_created rdf:resource="{$play-uri}"/>
+        {
+            for $author-idno in $author-idnos?*
+                return 
+                <crm:P14_carried_out_by rdf:resource="{$author-idno?uri}"/>
+        }
+    </rdf:Description>
+    
+    let $dracor-link := <rdfs:seeAlso rdf:resource="https://dracor.org/{$corpusname}/{$playname}"/>
+    
+    (: metrics :)
+    
+    let $averageClustering := if ($metrics/metrics/network/averageClustering/text() != "") then <dracon:averageClustering>{$metrics/metrics/network/averageClustering/text()}</dracon:averageClustering> else ()
```

Comment:
To shorten the following lines and make them easier to read and maintain I'd suggest to set `$metrics` to `doc($metricspath)/metrics/network` and wrap the lines, e.g.:

```
let $averageClustering := if ($metrics/averageClustering/text() != "")
  then <dracon:averageClustering>{$metrics/averageClustering/text()}</dracon:averageClustering>
  else ()
```